src/otx/algorithms/classification/adapters/mmcls/configurer.py

```diff
@@ -284,49 +222,25 @@ def configure_loss(self, cfg):
             )
             update_or_add_custom_hook(cfg, ib_loss_hook)
 
+    @staticmethod
+    def get_sampler_type(cfg):
+        """Return sampler type."""
+        if not cfg.model.get("multilabel", False) and not cfg.model.get("hierarchical", False):
+            cfg["task_adapt"].get("efficient_mode", True)
+            sampler_type = "balanced"
+        else:
+            cfg["task_adapt"].get("efficient_mode", False)
+            sampler_type = "cls_incr"
+        return sampler_type
```

Comment:
What are the `get()` in line#229 & 232 doing?

Reply:
They are unused codes, so I deleted them